.zenodo.json

```diff
@@ -0,0 +1,34 @@
+{
+  "license": {
+    "id": "BSD-3-Clause"
+  }, 
+  "creators": [
+    {
+      "name": "Alan Bromborsky"
+    },
+    {
+      "name": "Utensil Song"
+    },
+    {
+      "affiliation": "University of Cambridge",
+      "name": "Eric Wieser",
+      "orcid": "0000-0003-0412-4978"
+    },
+    {
+      "affiliation": "University of Cambridge",
+      "name": "Hugo Hadfield",
+      "orcid": "0000-0003-4318-050X"
+    },
+    {
+      "name": "{The Pygae Team}"
+    }
+  ],
+  "keywords": [
+    "python",
+    "clifford algebra",
+    "geometric algebra",
+    "computer algebra",
+    "geometry"
```

Comment:
Maybe add `symbolic`?

Reply:
```suggestion
    "symbolic algebra",
    "geometry"
```
or
```suggestion
    "symbolic",
    "geometry"
```
or did you have something else in mind?

Reply:
symbolic on it own would be enough. mixed with algebra will interfere with the search I guess

Reply:
I'll see which zenodo has more matches for

Reply:
![image](https://user-images.githubusercontent.com/64258/82572282-b3232700-9bb6-11ea-9b62-a409dd9e5da6.png)

It seems there's nothing more left. "Symbolic" is one particular feature of GAlgebra that is different from Clifford.

Reply:
https://zenodo.org/search?page=1&size=20&q=keywords:%22symbolic%22 has useless results, https://zenodo.org/search?page=1&size=20&q=keywords:%22symbolic%20algebra%22 looks at least vaguely sensible.

Reply:
I see that even SymPy didn't label itself with this...

Reply:
symbolic expression gives me some reasonable results

Reply:
Will add that later. Gave up yesterday because zenodo wouldn't load for me any more.